adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala

```diff
@@ -121,11 +113,14 @@ object FeatureRDD {
     // cache the rdd, since we're making multiple passes
     rdd.cache()
 
-    // aggregate to create the sequence dictionary
-    val sd = new SequenceDictionary(rdd.map(getSequenceRecord)
-      .distinct
-      .collect
-      .toVector)
+    // create sequence records with length max(start, end) + 1L
+    val sequenceRecords = rdd
+      .keyBy(_.getContigName)
+      .map(kv => (kv._1, Math.max(kv._2.getStart, kv._2.getEnd) + 1L))
```

Comment:
Small preference, but I'd rather `import scala.math.max` and change `Math.max` -> `max`.